misc/config.toml

```diff
@@ -42,6 +42,10 @@ anger = "496085813881470986"
 python = "490419105699069952"
 discordpy = "490419059964510210"
 
+# Cooldown after pinging a role in a channel, in seconds
+[pingcooldown]
+cooldown = "3600"
```

Comment:
Would put this in a different category, like `[moderation]` as `ping-cooldown`.

Reply:
Done.